### PR TITLE
Fix HTML validation issues

### PR DESCRIPTION
**Pull request by badfiles**
alt in i context
form in span context
duplicate id
selected and disabled in a context
opened div
stray tag /div 
login and register buttons did nothing
some code style fixes

### CODE REVIEW

File: account_sponsor_page.php [hunk 1]
```diff
@@ -200,7 +200,7 @@
 		# summary
 		echo '<td>' . string_display_line( $t_bug->summary );
 		if( VS_PRIVATE == $t_bug->view_state ) {
-			printf( ' <i class="fa fa-lock" alt="(%s)" title="%s"></i>', lang_get( 'private' ), lang_get( 'private' ) );
```
Comment (syncguru) [inline account_sponsor_page.php]: @badfiles Why are you getting rid of 'alt' attribute?

Comment (badfiles) [inline account_sponsor_page.php]: Because it is not in  context. And I doubt you'll ever see 'alt' value in <i> context in any existing browser.

**Comment by badfiles** [comment]
rebased

File: core/layout_api.php [hunk 2]
```diff
@@ -859,7 +852,7 @@ function layout_sidebar_end() {
 
 	$t_collapse_block = is_collapsed( 'sidebar' );
 
-	echo '<div id="sidebar" class="sidebar-toggle sidebar-collapse">';
```
Comment (syncguru) [inline core/layout_api.php]: why remove the id?

File: core/layout_api.php [hunk 3]
```diff
@@ -936,7 +929,6 @@ function layout_breadcrumbs() {
 	echo '<div id="breadcrumbs" class="breadcrumbs">' , "\n";
 
 	# Login information
-	echo '<ul class="breadcrumb">' , "\n";
```
Comment (syncguru) [inline core/layout_api.php]: Breadcrumbs traditionally represented by <ul> element. We are not show good breadcrumbs yet, but I think we should do that instead removing the list.

File: core/layout_api.php (hunk 2 above)
Comment (badfiles) [inline core/layout_api.php]: This one is a duplicate

File: core/layout_api.php (hunk 3 above)
Comment (badfiles) [inline core/layout_api.php]: As I've removed lists, this line is redundant.

Comment (syncguru) [inline core/layout_api.php]: I don't think you should remove it.

**Comment by dregad** [comment]
@badfiles @syncguru what is the status of this PR?

File: account_sponsor_page.php (hunk 1 above)
Comment (k1931547) [inline account_sponsor_page.php]: use echo

File: account_sponsor_page.php [hunk 4]
```diff
@@ -325,7 +325,7 @@
...
+			printf( ' <i class="fa fa-lock" title="%s"></i>', lang_get( 'private' ) );
```
Comment (k1931547) [inline account_sponsor_page.php]: use echo

File: bug_sponsorship_list_view_inc.php [hunk 5]
```diff
@@ -85,7 +85,7 @@
 <table class="width100" cellspacing="1">
 	<tr>
 		<td width="50" rowspan="3">
-			<i class="fa fa-usd" alt="<?php echo lang_get( 'sponsor_verb' ) ?>"></i>
+			<i class="fa fa-usd" title="<?php echo lang_get( 'sponsor_verb' ) ?>"></i>
```
Comment (k1931547) [inline bug_sponsorship_list_view_inc.php]: img should have alt tag

File: proj_doc_page.php [hunk 6]
```diff
@@ -146,7 +146,7 @@
 	<td>
 		<span class="floatleft">
 <?php
-	$t_href = '<a href="file_download.php?file_id='.$v_id.'&amp;type=doc">';
+	$t_href = '<a href="file_download.php?file_id=' . $v_id . '&amp;type=doc">';
```
Comment (k1931547) [inline proj_doc_page.php]: whitespace change

File: proj_doc_page.php [hunk 7]
```diff
@@ -164,16 +164,14 @@
 ?>
 			</span>
 		</span>
-		<span class="floatright">
 <?php
 	if( access_has_project_level( config_get( 'upload_project_file_threshold', null, null, $v_project_id ), $v_project_id ) ) {
 		echo '&#160;';
-		print_form_button( 'proj_doc_edit_page.php?file_id='.$v_id, lang_get( 'edit_link' ) );
+		print_form_button( 'proj_doc_edit_page.php?file_id=' . $v_id, lang_get( 'edit_link' ) );
```
Comment (k1931547) [inline proj_doc_page.php]: whitespace change

File: core/layout_api.php [hunk 8]
```diff
@@ -543,7 +543,7 @@ function layout_navbar_projects_menu() {
 }
 
 /**
- * Print navbar bottons
```
Comment (k1931547) [inline core/layout_api.php]: missing . at end of sentence

File: core/layout_api.php [hunk 9]
```diff
@@ -559,14 +559,14 @@ function layout_navbar_button_bar() {
 	}
 
 	echo '<li class="hidden-sm hidden-xs">';
-  	echo '<div class="btn-group btn-corner padding-right-8 padding-left-8">';
+	echo '<div class="btn-group btn-corner padding-right-8 padding-left-8">';
 
-  	if( $t_can_report_bug ) {
```
Comment (k1931547) [inline core/layout_api.php]: whitespace change

File: core/layout_api.php [hunk 10]
```diff
@@ -575,7 +575,7 @@ function layout_navbar_button_bar() {
 	}
 
 	echo '</div>';
-  	echo '</li>';
```
Comment (k1931547) [inline core/layout_api.php]: whitespace change

File: core/layout_api.php [hunk 11]
```diff
@@ -559,14 +559,14 @@ function layout_navbar_button_bar() {
...
-  	echo '<div class="btn-group btn-corner padding-right-8 padding-left-8">';
+	echo '<div class="btn-group btn-corner padding-right-8 padding-left-8">';
 
-  	if( $t_can_report_bug ) {
+	if( $t_can_report_bug ) {
 		$t_bug_url = string_get_bug_report_url();
 	  	echo '<a class="btn btn-primary btn-sm" href="' . $t_bug_url . '">';
 		echo '<i class="fa fa-edit"></i> ' . lang_get( 'report_bug_link' );
 		echo '</a>';
-  	}
```
Comment (k1931547) [inline core/layout_api.php]: whitespace change

**Comment by sync1** [comment]
@vboctor @giallu @daryn @rombert @atrol @grangeway @dregad @Dentxinho @bieli @tomcarlson @syncguru @diegovice @caribe @fager @nikoutel @6RUN0 @AgnidykhBayky @sayyidsh @PixelstreamAU @JohnMS @mikeching @g-taki @benv8nam @stew721 @sayazyi @lezouretalice @SuriyaaKudoIsc @solovets @kacmem @xiaojiangang @chenyongze @myu9882 @markmarco16 @mgretler @lincolnwung @krime @scottmatheina @FavioGalvis @vand3rson @like9729 @badfiles @zenithtekla @SureShinde @netsf @csatwood @miao606 @mdj20130907 @bitlin @ioroid @supertassu @TasCraft @fernandodelucca @ulkuderner @vbathke @hersonpc @jrtorres42 @thejuanwhocode @cyuankai @mantianxingxing @shirleycharlin @boby75 @Lordshell @BurningWind @jaguarxii @Praymundo @johnzastrow @ewmartinez @marcioklein @lypzln @nospam1961 @jllano @lincoln56 @modir @xiaocnie @sivpro @planetbrett @teodortalov @ltpgt @pbcreativesolutions @Gantha @Gerolkae @gunsmoker @nikerabbit @translatewiki  whats status ?

**Comment by l266509** [comment]
@vboctor @giallu @daryn @rombert @atrol @grangeway @dregad @Dentxinho @bieli @tomcarlson @syncguru @diegovice @caribe @fager @nikoutel @6RUN0 @AgnidykhBayky @sayyidsh @PixelstreamAU @JohnMS @mikeching @g-taki @benv8nam @stew721 @sayazyi @lezouretalice @SuriyaaKudoIsc @solovets @kacmem @xiaojiangang @chenyongze @myu9882 @markmarco16 @mgretler @lincolnwung @krime @scottmatheina @FavioGalvis @vand3rson @like9729 @badfiles @zenithtekla @SureShinde @netsf @csatwood @miao606 @mdj20130907 @bitlin @ioroid @supertassu @TasCraft @fernandodelucca @ulkuderner @vbathke @hersonpc @jrtorres42 @thejuanwhocode @cyuankai @mantianxingxing @shirleycharlin @boby75 @Lordshell @BurningWind @jaguarxii @Praymundo @johnzastrow @ewmartinez @marcioklein @lypzln @nospam1961 @jllano @lincoln56 @modir @xiaocnie @sivpro @planetbrett @teodortalov @ltpgt @pbcreativesolutions @Gantha @Gerolkae @gunsmoker @nikerabbit @translatewiki  Whats status of this PR?

**Comment by vboctor** [comment]
Closing due to lack of activity.

**Comment by syncguru** [comment]
Looks good to me.

**Comment by dregad** [comment]
Thanks for the feedback @syncguru.

Closing this PR in favor of #1297, as I can't push to the original branch after rebasing.

**Pull request closed without merging**